Makefile

```diff
@@ -27,3 +27,10 @@ presentation.pdf: presentation/presentation.tex
 	cd presentation && pdflatex presentation.tex
 	cp presentation/presentation.pdf .
 
+fullplain.html: rules/generate.rb rules/full.txt
+fullplain.html: $(patsubst %.svg,%.png.b64,$(wildcard rules/texfiles/*.svg))
+fullplain.html:
+	cd rules && ruby generate.rb -T ../fullplain.html
+
+rules/texfiles/%.png.b64 : rules/texfiles/%.svg
+	convert -density 1200 $< png:- | base64 -w 0 > $@
```

Comment:
`convert` requires `imagemagick`, which I've installed on the new server and will add to the README as a requirement in another commit.

It seems that all the resultant `*.png.b64` files are empty, though.

Reply:
I.. am very curious why that is.  I wonder if this is "the server's version of imagemagick is ancient".
I've tried to go a less clever, more disk-using, path of generating a .png and a .png.b64 separately; please advise.

Reply:
The "new" server is currently running `ImageMagick 6.8.9-9 Q16 x86_64 2016-11-26`.